Server/config/env/template.env

```diff
@@ -0,0 +1,3 @@
+
+
+WEB_CLIENT_HOST=YOUR_CLIENT_URL
```

Comment:
Notice the naggy red circle at the end of the line here. It indicates one should make sure to have `insert_final_newline = true` configured in the EditorConfig!

It is a bit hard to understand what goes in here also, without digging into the server implementation details. Maybe `http://localhost:4200` would be a sensible default?